src/react/components/pages/projectSettings/projectMetrics.tsx

```diff
@@ -210,7 +210,14 @@ export default class ProjectMetrics extends React.Component<IProjectMetricsProps
      *   Note: video frames are not counted, only the video container
      */
     private getSourceAssetCount = () => {
-        const assets = this.state.sourceAssets;
+        const assets = this.state.projectAssetsMetadata.map((e) => e.asset.name);
```

Comment:
Consider using JS `Set<IAsset>` here to help deal with duplicates.  You would avoid having to check for existence before adding assets.